# Preeria_caryophylla

The main task of the project was assembly the genome of the bacterium Preeria Caryophilla and then analyze it.

Goals of the project:

1. Preprocess data
2. Assembly genome
3. Genome annotation 
4. Search for orthologs between Holosporales representatives

In this project we used next programms:
1. Flye (v2.8.2) - genome assembler.
2. Kaidju (v1.7.3) - is a program for the taxonomic classification of high-throughput sequencing reads.
3. Minimap2 (v2.17) - sequence alignment.
4. Quast (v5.0.2) - QUality ASsessment Tool.
5. Proteinortho (v6) - is a tool to detect orthologous genes within different species.
6. Pilon (v1.2.3) - improvement of genomic assembly, bug fix.
7. Nanoflit (v2.5.0) - oxford nanopore data preprocessing.
8. BWA(v0.7.17) - reed mapping.
9. Prokka (v1.14.5) - genome annotation.
10. Prodigal (v2.6.3) - gene prediction for prokaryotic genomes.
11. Diamond (v4.0.515) - sequence aligner for protein.
12. OrthoVenn2
13. Racon (v1.4.3) - standalone consensus module to correct raw contigs generated by rapid assembly methods.
14. Medaka (v1.2.0) - is a tool to create consensus sequences and variant calls from nanopore sequencing data.
